src/foam/dao/EasyDAO.js

```diff
@@ -227,6 +303,12 @@ return delegate;
       generateJava: false,
       value: false
     },
+    {
+      /** Enable standard authorization. */
```

Comment:
Yes, currently it will be only authorizing objects that implement authorizable but the StandardAuthorizer is in progress so I think we can keep it this way. Troy is correct(almost) @noodlemoodle is working on it.